### PR TITLE
fix: add column v6, v7

### PR DESCRIPTION
**Pull request by HaoLiu97**
Fix: https://github.com/casbin/gorm-adapter/issues/151

### CODE REVIEW

**Comment by HaoLiu97** [comment]
need to change unit test

**Comment by hsluoyz** [comment]
@HaoLiu97 fix:

image

**Comment by tangyang9464** [comment]
@HaoLiu97 plz format your code (``go fmt``)

File: adapter.go [hunk 1]
```diff
@@ -49,6 +49,8 @@ type CasbinRule struct {
 	V3    string `gorm:"size:100"`
 	V4    string `gorm:"size:100"`
 	V5    string `gorm:"size:100"`
+	V6    string `gorm:"size:25"`
```
Comment (tangyang9464) [inline adapter.go:52]: size:100 instead of 25 to be consistent

**Review by tangyang9464 — CHANGES_REQUESTED** [review]

File: adapter.go (hunk 1 above)
Comment (HaoLiu97) [reply adapter.go:52]: Set the size of V6 and V7 as 100 will make the length of unique index exceed the mysql default limitation, which is 3072 bytes.

Comment (hsluoyz) [reply adapter.go:52]: @tangyang9464 using 100 will cause index overflow. So he uses 25 here as a workaround.

**Pull request merged**